projects/components/src/stepper/tab-controls/stepper-tab-controls.component.ts

```diff
@@ -0,0 +1,23 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ContentHolder, CONTENT_HOLDER_TEMPLATE } from '../../content/content-holder';
+import { StepperComponent } from '../stepper.component';
+import { StepperTabComponent } from '../tab/stepper-tab.component';
+
+@Component({
+  selector: 'ht-stepper-tab-controls',
+  template: CONTENT_HOLDER_TEMPLATE,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class StepperTabControlsComponent extends ContentHolder {
+  public constructor(public stepper: StepperComponent, public step: StepperTabComponent) {
+    super();
+  }
```

Comment:
do we need the parent elements to be public?

Reply:
Not right now. Either we expose APIs like we did below for `next` and `previous` or allow access to stepper which gives more access to the custom controls.

Reply:
Marked as `private`